nengo/spa/basalganglia.py

```diff
@@ -141,3 +143,36 @@ def add_dot_input(self, index, source, symbol, scale):
         with self.spa:
             nengo.Connection(output, self.input[index:index+1],
                              transform=transform, synapse=self.input_synapse)
+
+    def add_scalar_input(self, index, source):
+        """ Add a scalar input that will vary over time.
+
+        This would be used for such a thing as the ouput of the Compare module.
+
+        Parameters
+        ----------
+        index : int
+            the index of the action
+        source : Source
+            the module output to read from
+        symbol : Source
+            the semantic pointer to compute the dot product with
+        scale : float
+            a scaling factor to be applied to the result
+        """
+        output, _ = self.spa.get_module_output(source.name)
+        if(output.size_out != 1):
```

Comment:
> A quick Google gave me nothing and flake8 didn't set off any alarms, so I guess it's personal? I'm fine with noting it down on our personal style guide or even putting it into our flake8 checks.

Hmm, all of the single-line python if statements I can find (in nengo or in other places) do not use parentheses.  Do you have examples where parentheses are used?  I'm curious what they are...